.config/dotnet-tools.json

```diff
@@ -9,7 +9,7 @@
       ]
```

Comment:
I think you can remove `fake-cli` from this file now?

Reply:
these are dotnet tools for building the template itself - currently still using fake tool here.
I've removed fake tool from the default template though.